shared/src/main/scala/tastyquery/ast/Symbols.scala

```diff
@@ -37,29 +55,40 @@ object Symbols {
     def unapply(s: Symbol): Option[Name] = Some(s.name)
   }
 
-  final class RegularSymbol(override val name: Name, override val owner: Symbol) extends Symbol(name, owner)
+  final class RegularSymbol(override val name: Name, owner: Symbol | Null) extends Symbol(name, owner)
 
-  abstract class DeclaringSymbol(override val name: Name, override val owner: Symbol) extends Symbol(name, owner) {
+  abstract class DeclaringSymbol(override val name: Name, rawowner: Symbol | Null) extends Symbol(name, rawowner) {
     /* A map from the name of a declaration directly inside this symbol to the corresponding symbol
      * The qualifiers on the name are not dropped. For instance, the package names are always fully qualified. */
-    protected val myDeclarations: mutable.HashMap[Name, Symbol] = mutable.HashMap[Name, Symbol]()
 
-    def addDecl(decl: Symbol): Unit = myDeclarations(decl.name) = decl
-    def getDecl(name: Name): Option[Symbol] = myDeclarations.get(name)
-    def declarations: List[Symbol] = myDeclarations.values.toList
+    protected val decls: mutable.MultiDict[Name, Symbol] = mutable.MultiDict.empty
```

Comment:
This seems to be the only real usage of scala-collection-contrib. Is it really worth the addition of that dependency, as opposed to using a `mutable.Map[Name, mutable.Set[Symbol]]`?

Reply:
I also use it to sort packages in ClassLoaders

Reply:
removed all usages